x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/http/TemplateHttpResource.java

```diff
@@ -31,6 +36,15 @@
 
     private static final Logger logger = LogManager.getLogger(TemplateHttpResource.class);
 
+    private static final Map<String, String> PARAMETERS;
+
+    static {
+        Map<String, String> parameters = new HashMap<>(
+            PublishableHttpResource.RESOURCE_VERSION_PARAMETERS);
+        parameters.put(INCLUDE_TYPE_NAME_PARAMETER, "true");
+        PARAMETERS = Collections.unmodifiableMap(parameters);
```

Comment:
nit: I think you can use https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/common/collect/MapBuilder.java#L87 to shorten this up abit.

Reply:
Nice, I didn't realize we had that.